api/osb/osb_store_plugin.go

```diff
@@ -433,6 +433,12 @@ func (sp *storePlugin) Deprovision(request *web.Request, next web.Handler) (*web
 	if err != nil {
 		return nil, err
 	}
+	ctx := request.Context()
+	_, operationFound := opcontext.Get(ctx)
+	if operationFound {
+		log.C(ctx).Debug("operation found in context, continue..")
```

Comment:
not clear why continue, can you enhance the message (as the one below about polling)?